### PR TITLE
Merge Startup.cs logic into Program.cs.

### PR DESCRIPTION
**Pull request by LayersOfAbstraction**
Hi there, 

I deleted Startup.cs and merged the logic from that file into Program.cs with the minimal hosting model as per #79.

Confirmed as running in IIS. Was able to log in to the account profile with ClientID and Domain upon compilation. As discussed I will try to update the docs to reflect this so we can merge everything at the same time.

### CODE REVIEW

File: Quickstart/Sample/Program.cs
Comment (frederikprijck) [inline Quickstart/Sample/Program.cs:38]: Any reason for adding this while we didn't have this before?

File: Quickstart/Sample/Program.cs
Comment (frederikprijck) [inline Quickstart/Sample/Program.cs:17]: I'm not getting any issues when removing those two. Can you elaborate?

File: Quickstart/Sample/Program.cs
Comment (frederikprijck) [inline Quickstart/Sample/Program.cs:19]: I'm not getting any issue when removing this, can you elaborate?

**Review by frederikprijck — CHANGES_REQUESTED** [review]
A couple of changes. I would like to ask to stick to moving things, and not adding different things.

If you believe these are useful additions, we can always track them through a different PR, but it complicates reviewing this PR.

File: Quickstart/Sample/Program.cs
Comment (LayersOfAbstraction) [reply Quickstart/Sample/Program.cs:38]: No real reason for that. I just used it because that was what was in the [MS ASP.NET 6 code samples](https://learn.microsoft.com/en-us/aspnet/core/migration/50-to-60?view=aspnetcore-7.0&tabs=visual-studio#:~:text=hsts.%0A%20%20%20%20app.UseHsts()%3B%0A%7D-,app.UseHttpsRedirection()%3B,-app.UseStaticFiles()%3B%0A%0Aapp) from migrating to the minimal hosting model. I'm getting rid of that. Taking it out on my end resulted in no changes.

File: Quickstart/Sample/Program.cs
Comment (LayersOfAbstraction) [reply Quickstart/Sample/Program.cs:17]: Sure, so I've been getting an error similar to #37778. 

It happened on the lines

```csharp
endpoints.MapDefaultControllerRoute();
//AND
app.UseAuthorization();
```
The only way I resolved it was by 
[watching this tutorial from where it starts ](https://www.youtube.com/watch?v=vhNhcuht0J0&t=385s) until 6:56. Sadly I cannot any long replicate the error. 

Tried everything I could think of to replicate it. Rebuilt the binaries, Restarted the computer. I suppose I hadn't rebuilt the project properly when I had that error? So I'm following through with your suggested changes. Moving on to the docs, thanks.

Comment (LayersOfAbstraction) [reply Quickstart/Sample/Program.cs]: Whoops forgot to take ```builder.Services.AddMvc();``` out. Will do that now as that line is no longer resulting in the error which is really really strange. Before that, it fixed an error called.
[No service for type 'Microsoft.AspNetCore.Mvc.ViewFeatures.ITempDataDictionaryFactory' has been registered](https://stackoverflow.com/questions/38709538/no-service-for-type-microsoft-aspnetcore-mvc-viewfeatures-itempdatadictionaryfa) the suggested solution worked.

Comment (frederikprijck) [reply Quickstart/Sample/Program.cs]: Thanks!

**Comment by frederikprijck** [comment]
Thanks, looks good. Will approve once we have the docs, to avoid merging this accidentically.

**Review by frederikprijck — APPROVED** [review]

**Pull request merged**
